src/javascript/app/pages/form/barrier-fields.jsx

```diff
@@ -0,0 +1,137 @@
+import React, { useState, useEffect } from 'react';
+import {
+    TextField,
+    Tooltip,
+} from '@deriv-com/quill-ui';
+// eslint-disable-next-line import/no-unresolved
+import { StandaloneCircleInfoRegularIcon } from '@deriv/quill-icons/Standalone';
+import { useBarrierChange } from '../../hooks/events.js';
+import dataManager from '../../common/data_manager.js';
+import { localize } from '../../../_common/localize.js';
+import Defaults, { PARAM_NAMES } from '../trade/defaults.js';
+
+const BarrierFields = ({ formName, handleAmountChange }) => {
+    const [barrierData, setBarrierData] = useState();
+    const hasBarrierChange = useBarrierChange();
+
+    const barrier = Defaults.get(PARAM_NAMES.BARRIER);
+    const barrier_high = Defaults.get(PARAM_NAMES.BARRIER_HIGH);
+    const barrier_low = Defaults.get(PARAM_NAMES.BARRIER_LOW);
```

Comment:
both camelCase and snake_case case are being used for field names. Could we follow our guideline and use snake_case for all field names? - https://github.com/deriv-com/smarttrader/blob/master/src/javascript/README.md